release_notes_generator/data/miner.py

```
@@ -350,3 +358,15 @@ def __filter_duplicated_issues(data: MinedData) -> "MinedData":
 
         data.issues = filtered_issues
         return data
+
+    def _fetch_prs_for_fetched_cross_issues(self, issues: dict[Issue, Repository], data: MinedData) -> dict[str, list[PullRequest]]:
```

Ruff (0.13.3)

Unused method argument: `data`

(ARG002)